craft_parts/parts.py

```diff
@@ -388,9 +389,12 @@ class PartSpec(BaseModel):
         default=None,
         description="The commands to run instead of the default behavior of the pull step.",
         examples=[
-            """|
-              craftctl default
-              rm $CRAFT_PART_SRC/pyproject.toml"""
+            textwrap.dedent(
+                """\
+                |
+                craftctl default
+                rm $CRAFT_PART_SRC/pyproject.toml"""
+            )
```

Comment:
Could we still indent starting at the second line?

```suggestion
            textwrap.dedent(
                """\
                |
                  craftctl default
                  rm $CRAFT_PART_SRC/pyproject.toml"""
            )
```
Kitbash will pass along PyYAML's complaints if it's left as is.

Reply:
how does kitbash render this^ version?

Reply:
With the added indentation, it renders as:

<img width="822" height="128" alt="image" src="https://github.com/user-attachments/assets/60327160-f145-4b96-80eb-26ab9dfb5828" />

No PyYAML errors either :+1: